src/utils/Enums.ts

```diff
@@ -232,3 +232,13 @@ export enum StringKey {
   welcomeMessageLiveWelcomeStepContent = constants.welcomeMessageLiveWelcomeStepContent,
   welcomeMessageLiveWelcomeStepTitle = constants.welcomeMessageLiveWelcomeStepTitle,
 }
+
+export enum LaunchType {
+  hot = constants.hot,
+  cold = constants.cold,
+  unknown = constants.unknown,
```

Comment:
Let's add a comment to state that `LaunchType.unknown` is iOS only as well.
```suggestion
  /**
    * iOS only
    */
  unknown = constants.unknown,
```